sqlalchemy_imageattach/entity.py

```diff
@@ -127,8 +127,16 @@ def image_attachment(*args, **kwargs):
        implementation.
 
     """
+    if 'has_many' in kwargs:
```

Comment:
That confliction exactly was my intention.  If `uselist=False` is passed to `relationship()`, SQLAlchemy-ImageAttach does not work as it is intended, because it would always return a random single image.  So my opinion is that `realtionship()`’s own `uselist=False` should be opaque with `image_attachment()`.